sktime/annotation/tests/test_e_agglo.py

```diff
@@ -9,7 +9,7 @@
 from sktime.annotation.e_agglo import EAGGLO
 
 
-def test_default_params_univariate():
+def test_fit_default_params_univariate():
     """Test data."""
     X = pd.DataFrame([-7.207066, -5.722571, 5.889715, 5.488990])
```

Comment:
add a comment explaining where these numbers (and where teh expected numbers) come from.